urlResolvers/github.js

```diff
@@ -0,0 +1,42 @@
+const URLResolver = require('./urlResolver');
+const GitHubAPIHandler = require('../sourceHandlers/github');
+
+const stripHttp = /^(http(s)?:\/\/)?(www.)?/;
```

Comment:
Good catch.  Copied wrongly from the deleted code.  I'm not super worried about matching the empty string.